src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(operand) {
+      this.result += operand;
+
+      return this;
+    },
+    subtract(operand) {
+      this.result -= operand;
+
+      return this;
+    },
+    multiply(operand) {
+      this.result *= operand;
+
+      return this;
+    },
+    divide(operand) {
+      if (operand !== 0) {
+        this.result /= operand;
+      }
```

Comment:
Consider throwing an error or returning a specific value when attempting to divide by zero, as simply not performing the operation might not provide clear feedback to the user of the calculator about what went wrong.